api/src/utils/csv-utils/csv-config-utils.ts

```diff
@@ -129,6 +129,22 @@ export class CSVConfigUtils<StaticHeaderType extends Uppercase<string> = Upperca
     this._config.staticHeadersConfig[header] = { ...this._config.staticHeadersConfig[header], ...headerConfig };
   }
 
+  /**
+   * Set all static header configs. Injects the header configs into the CSV static headers config.
+   * Similar to `setStaticHeaderConfig` but for all headers.
+   *
+   * @param {Record<StaticHeaderType, CSVHeaderConfig>} headersConfig - The header configs
+   * @returns {void}
+   */
+  setAllStaticHeaderConfigs(headersConfig: Record<StaticHeaderType, CSVHeaderConfig>): void {
```

Comment:
Could this function just call the single header version (`setStaticHeaderConfig`) in a loop?